code/game/verbs/who.dm

```diff
@@ -60,7 +60,7 @@
 	to_chat(src, msg)
 
 /client/verb/staffwho()
-	set category = "Staffhelp"
+	set category = "Admin"
```

Comment:
Ehhh, this lists *all* staff, not just admins. Unless you murdered the `Staffhelp` tab I'd recommend keeping it where it is.